README.md

```diff
@@ -79,16 +79,16 @@ using HiveMQtt.MQTT5.Types;
 
 // Setup Client options and instantiate
 var options = new HiveMQClientOptionsBuilder().
-                    WithBroker('candy.x39.eu.hivemq.cloud').
+                    WithBroker("candy.x39.eu.hivemq.cloud").
                     WithPort(8883).
-                    WithUseTLS(true).
+                    WithUseTls(true).
```

Comment:
It actually is camel case right now even though it shouldn't be.  I'll fix this today.